client-core/src/wallet/syncer.rs

```diff
@@ -225,7 +222,7 @@ struct WalletSyncerImpl<'a, S: SecureStorage, C: Client, D: TxDecryptor> {
 
 impl<'a, S: SecureStorage, C: Client, D: TxDecryptor> WalletSyncerImpl<'a, S, C, D> {
     fn new(env: &'a WalletSyncer<S, C, D>) -> Result<Self> {
-        let wallet = service::load_wallet(&env.storage, &env.name, &env.passphrase)?
+        let wallet = service::load_wallet(&env.storage, &env.name, &env.enckey)?
```

Comment:
I don't think this is a good pattern to have. Ideally we should use `Service` objects and their functions. It's ok for now.

Reply:
This is debatable, we can discuss this some time.